src/screens/OtherProfile/index.js

```diff
@@ -818,7 +814,11 @@ const styles = StyleSheet.create({
     display: 'flex',
     flexDirection: 'row',
     marginBottom: 20
-  }
+  },
+  blockBtnStyle: (isAnonym) => ({
+    backgroundColor: isAnonym ? COLORS.anon_primary : COLORS.signed_primary,
+    paddingLeft: 0
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of a new `blockBtnStyle` property in the styles object based on user anonymity is correctly done. However, as with previous comments, please ensure these color changes do not negatively affect the accessibility of the application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->